packages/tools/src/tools/annotation/DragProbeTool.ts

```diff
@@ -155,16 +155,36 @@ class DragProbeTool extends ProbeTool {
 
     const color = this.getStyle('color', styleSpecifier, annotation);
 
+    const isPreScaled = isViewportPreScaled(viewport, targetId);
+
+    const isSuvScaled = this.isSuvScaled(
+      viewport,
+      targetId,
+      annotation.metadata.referencedImageId
+    );
+
     if (!data.cachedStats[targetId]) {
       data.cachedStats[targetId] = {
         Modality: null,
         index: null,
         value: null,
       };
 
-      this._calculateCachedStats(annotation, renderingEngine, enabledElement);
+      this._calculateCachedStats(
+        annotation,
+        renderingEngine,
+        enabledElement,
+        isPreScaled,
+        isSuvScaled
```

Comment:
can we add them as object at the end as `options`? instead of positional arguments?